Hover/Model/HoverItem.swift

```diff
@@ -13,11 +13,11 @@ public struct HoverItem {
     
     // MARK: Properties
     let title: String?
-    let image: UIImage
+    let image: UIImage?
     let onTap: () -> ()
     
     // MARK: Lifecycle
-    public init(title: String? = nil, image: UIImage, onTap: @escaping () -> ()) {
+    public init(title: String? = nil, image: UIImage?, onTap: @escaping () -> ()) {
```

Comment:
Could you explain why we should change this? I don't think it makes much sense to update this since without an Image the UI wouldn't make much sense. 🤔

Reply:
It made sense to me as in HoverConfiguration the image is already an optional. It doesn't matter if the image is nil, it keeps working and it lets the users interact with the buttons besides the image is not showing. To keep consistency in the code, I would use optionals for both Configuration and Item.
In terms of UI it doesn't make much sense not to have an image displayed in the button, as you said. Maybe Hover could provide a default image for these cases.

PS: If this is finally not changed, the readme should be updated accordingly (images values should be unwrapped in HoverItem). Just my 2 cents 😇

Reply:
> It made sense to me as in HoverConfiguration the image is already an optional.

This was definitely not intended when it was implemented back in the day. This should have been non-optional.

> To keep consistency in the code, I would use optionals for both Configuration and Item.

We could either make both option or non-optional. Since updating `HoverConfiguration` to be non-optional would make it a breaking change, I'd rather make both optionals to at least keep it consistent 👍 

> In terms of UI it doesn't make much sense not to have an image displayed in the button, as you said. Maybe Hover could provide a default image for these cases.

Let's keep it simple for now.